roles/network-multus/tasks/provision.yml

```diff
@@ -46,24 +46,49 @@
 
   - name: Create multus Resources
     shell: "{{ cluster_command }} apply -f /tmp/multus.yml"
+  when: crd.stdout=="0"
 
+- name: Deploy CNI plugins
+  block:
   - name: Render cni plugins deployment yaml
     template:
       src: cni-plugins.yml
       dest: /tmp/cni-plugins.yml
 
   - name: Create cni plugins Resources
     shell: "{{ cluster_command }} apply -f /tmp/cni-plugins.yml"
+  when: deploy_cni_plugins
 
+- name: Deploy OVS Plugin
+  block:
+  - name: Render OVS deployment yaml
+    template:
+      src: ovs-cni.yml
+      dest: /tmp/ovs-cni.yml
 
-  - name: Wait until multus is running
-    shell: "{{ cluster_command }} -n {{ namespace }} get daemonset | grep {{ item }} | awk \'{ if ($3 == $4) print \"0\"; else print \"1\"}\'"
-    loop:
-    - kube-multus-ds-amd64
-    - kube-cni-plugins-ds-amd64
-    register: result
-    until: result.stdout == "0"
-    retries: 20
-    delay: 5
+  - name: Create ovs Resources
+    shell: "{{ cluster_command }} apply -f /tmp/ovs-cni.yml"
+  when: deploy_ovs_plugin
 
-  when: crd.stdout=="0"
+- name: Wait until multus is running
+  shell: "{{ cluster_command }} -n {{ namespace }} get daemonset | grep {{ multus_provisioner_name }} | awk \'{ if ($3 == $4) print \"0\"; else print \"1\"}\'"
```

Comment:
@leongold Can you please break this line and similar in this PR?

Reply:
Done